pkg/apis/minio.min.io/v2/types.go

```diff
@@ -390,6 +390,22 @@ const (
 	HealthStatusRed HealthStatus = "red"
 )
 
+// TenantUsage are metrics regarding the usage and capacity of the tenant
+type TenantUsage struct {
+	// Capacity the usage capacity of this tenant.
+	// +optional
+	Capacity int64 `json:"capacity,omitempty"`
+	// Capacity the raw capacity of this tenant.
+	// +optional
+	RawCapacity int64 `json:"rawCapacity,omitempty"`
+	// Usage is how much data is managed by MinIO
+	// +optional
+	Usage int64 `json:"usage,omitempty"`
+	// Usage is the raw usage on disks
+	// +optional
+	RawUsage int64 `json:"rawUsage,omitempty"`
```

Comment:
Is this capacity in bytes? we can add some comments for developers to know what to expect

Reply:
done